### PR TITLE
Fix some typos

### PR DESCRIPTION
**Pull request by Kijek3**

### CODE REVIEW

File: src/_locales/pl.config [hunk 1]
```diff
@@ -96,10 +96,10 @@ Zastosuj ustawienia tylko dla bieżącej strony
 Lista stron
 
 @invert_listed_only
-Odwórć tylko list.
+Odwróć tylko listę
 
 @not_invert_listed
-Nie odwracaj list.
+Nie odwracaj listy
```
Comment (Gusted) [inline src/_locales/pl.config:102]: Missing a . after the sentence?

File: src/_locales/pl.config [hunk 2]
```diff
@@ -96,10 +96,10 @@ Zastosuj ustawienia tylko dla bieżącej strony
 Lista stron
 
 @invert_listed_only
-Odwórć tylko list.
+Odwróć tylko listę
```
Comment (Gusted) [inline src/_locales/pl.config:99]: Missing a . after the sentence?

File: src/_locales/pl.config (hunk 1 above)
Comment (Kijek3) [reply src/_locales/pl.config:102]: That dot was previously added only for shorthand purposes. "list" in polish is a plural form, so it doesn't "fit" good. "listy" is full single form and don't need that dot - it was made only for shorthand, not to finish a sentence.

File: src/_locales/pl.config (hunk 2 above)
Comment (Kijek3) [reply src/_locales/pl.config:99]: Same here. That dot was previously added only for shorthand purposes. "list" in polish is a plural form, so it doesn't "fit" good. "listę" is full single form and don't need that dot - it was made only for shorthand, not to finish a sentence.

**Pull request merged**